packages/xo-web/src/xo-app/backup/new/index.js

```diff
@@ -610,7 +610,7 @@ const New = decorate([
       },
       setMaxExportRate({ setGlobalSettings }, rate) {
         setGlobalSettings({
-          maxExportRate: rate * (1024 * 1024),
+          maxExportRate: rate !== undefined ? rate * (1024 * 1024) : undefined,
```

Comment:
What are the consequences of this bug? Do we need to release a patch?

Reply:
The consequences are quite clear in the changelog, my bad :slightly_smiling_face:

Reply:
Yes it does.

This is probably an acceptable work-around until the next release.